app/policies/application_policy.rb

```diff
@@ -6,6 +6,14 @@ class ApplicationPolicy < ActionPolicy::Base
 
   private
 
+  def administors_collection?(collection)
+    administrator? || depositor? || reviews_collection?(collection) || manages_collection?(collection)
+  end
+
+  def reviews_collection?(collection)
+    allowed_to?(:review?, collection)
+  end
+
```

Comment:
This method was basically the same in both work_policy and work_version_policy, putting it here in the same way the manages_collection? method below is here.